rules.go

```diff
@@ -104,7 +103,11 @@ func IsContainWhitespace(str string) bool {
 }
 
 func IsCountry(str string) bool {
-	_, err := gountries.New().FindCountryByAlpha(str)
+	var err error
+	if gountriesQuery != nil {
```

Comment:
if gountriesQuery nil, this func will always return `true`, right? I think if gountriesQuery is nil, we should consider it as an error, so it always return false. 
Or we can also init here if gountriesQuery is nil. What do you think?

Reply:
yes, lazy init, i like that

Reply:
sure, let me add lazy initialization.